examples/sample-app/README.md

```diff
@@ -118,6 +137,8 @@ the present working directory is the same directory as this README.
       you'll see the page update as you deploy objects into the project
       and run builds.
 
+         **VAGRANT USERS**:  You can view the page at `https://<host>:8444` but the web console will not work correctly.
```

Comment:
Without an ssh tunnel, this will not work at all; I would change this to:

    **VAGRANT_USERS**: The web console is not available outside of the Vagrant box; move on to the next step.